JokeRandomizer/JokeRandomizer/CustomButtons/FavoriteButton.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  FavoriteButton.swift
+//  JokeRandomizer
+//
+//  Created by john.adrian.de.claro on 12/2/22.
+//
+
+import UIKit
+
+
+class AddFavoriteButton: UIButton {
+    
+    private let addFavoriteButton: UIButton = {
+            let favButton = UIButton()
+            favButton.translatesAutoresizingMaskIntoConstraints = false
+            let config = UIImage.SymbolConfiguration(pointSize: 32.0, weight: .bold)
+            let image = UIImage(systemName: "plus", withConfiguration: config)
+            favButton.setImage(image, for: .normal)
+            favButton.tintColor = .white
+            favButton.backgroundColor = .blue
+            favButton.layer.cornerRadius = 10
+            favButton.layer.maskedCorners = [.layerMinXMinYCorner, .layerMaxXMaxYCorner]
+            return favButton
+        }()
+    
+    func shakeFavorite() {
```

Comment:
I noticed this function exist in CustomButton as well. Please try to make a generic button that can be configured for both requirements. This enforces your code to adheare to the DRY(Don't repeat yourself) principle.

Reply:
Delete FavoriteButton.swift move to custom button